stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -1952,3 +1952,13 @@ function fill!(A::Union{SparseVector, SparseMatrixCSC}, x)
     end
     return A
 end
+
+
+kron(v::SparseVector,A::SparseMatrixCSC) = kron(SparseMatrixCSC(v),A)
+kron(A::SparseMatrixCSC,v::SparseVector) = kron(A,SparseMatrixCSC(v))
+
+function kron(v1::SparseVector,v2::SparseVector)  
```

Comment:
Thanks for the PR @lruthotto. A couple of comments: 
- A `kron` method for two `SparseVector`s is already defined at https://github.com/JuliaLang/julia/blob/521dec9bf4990a0ec0fb827e1167e6c5195d392d/stdlib/SparseArrays/src/linalg.jl#L825

- I also think that's the proper place to put the methods for one sparse vector and one sparse matrix.

Reply:
That does seem more appropriate.